### PR TITLE
Add Phaser

### PR DESCRIPTION
**Pull request by leereilly**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Phaser is awesome, has about [16k ★s](https://github.com/photonstorm/phaser/stargazers) , and there are [hundreds of repos tagged with `phaser`](https://github.com/topics/phaser) 💜

### CODE REVIEW

**Comment by leereilly** [comment]
Whatcha think about bumping the value for this 🤔 https://github.com/github/explore/blob/2a8dcde4db07f062372ecfd55cde2d27be1872ef/test/test_helper.rb#L9

I'm no image optimization specialist, but I'm finding it difficult to drop this image below ~ 72000 bytes and keeping the quality...

**Comment by cheshire137** [comment]
I think it's fine to bump the file size limit!

File: topics/phaser/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+aliases: phaserjs
+created_by: Richard Davey
+display_name: Phaser
+github_url: https://github.com/photonstorm/phaser
+logo: phaser.png
+short_description: Phaser is a fun, free and fast 2D game framework for making HTML5 games for desktop and mobile web browsers, supporting Canvas and WebGL rendering.
```
Comment (nayafia) [inline topics/phaser/index.md]: Think this is a wee bit long. Can you shorten down to ~75 chars?

Comment (cheshire137) [reply topics/phaser/index.md]: In #86, I added a test to check the length of short_description. I also bumped the length to 130 characters based on how long existing topic short_descriptions are.

Comment (cheshire137) [reply topics/phaser/index.md]: If we did want to shorten this, I think this reads well: "Phaser is a fun, free and fast 2D game framework for making HTML5 games for desktop and mobile web browsers."

**Comment by styline** [comment]
: control>.data.systems.exe;
: item.folders.x.exe
template.on. demand
: run.exe
: save.close

**Comment by styline** [comment]
: callcenter.days.exe
loading.histdata. registre folders;
open source md
;))))
java sun program.exe
run time
save clos

**Comment by MikeMcQuaid** [comment]
Thanks @leereilly!

**Pull request merged**